codegen/protocol-test-codegen-local/build.gradle.kts

```diff
@@ -41,8 +41,12 @@ val codegenTests = listOf(
         "Waiters"
     ),
     CodegenTest(
-        "aws.protocoltests.restjson#TestService",
+        "aws.protocoltests.eventstream#TestService",
         "EventStream"
+    ),
+    CodegenTest(
+        "aws.protocoltests.eventstream#RPCTestService",
+        "RPCEventStream"
     )
```

Comment:
Code generate the non-RPC EventStream & RPC EventStream models with `protocol-test-codegen-local`.